PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskRawJetWithEP.h

```diff
@@ -168,7 +168,7 @@ class AliAnalysisTaskRawJetWithEP : public AliAnalysisTaskEmcalJet {
   // ### private ###############################################################
   private:
     
-    AliAODEvent* fAOD;                //!<! AOD event
+    AliAODEvent* fAOD;                /// AOD event
```

Comment:
You're trying to stream a nullptr. If this works, it will only work coincidentally (I guess it will work, but I don't see how this would help)